src/components/date-range-picker/date-range-picker.spec.ts

```diff
@@ -102,10 +103,45 @@ describe('Date Range Picker', () => {
         expect(getDate(1)).toBeNull();
     });
 
+    it('should allow a half time zone to be set if it is present in the timezone array', async() => {
+        component.showTime = true;
+        component.showTimezone = true;
+        component.timezones = [{ name: 'GMT-3.5', offset: 210 }, { name: 'GMT-3', offset: 180 }, { name: 'GMT-2', offset: 120 }, { name: 'GMT-1', offset: 60 }, { name: 'GMT', offset: 0 }];
+        component.startTimezone = { name: 'GMT-3.5', offset: 210 };
+        component.endTimezone = { name: 'GMT-3.5', offset: 210 };
+
+        fixture.detectChanges();
+        await fixture.whenStable();
+
+        expect(getGMTValue().getAttribute('ng-reflect-value')).toBe('GMT-3.5');
+        expect(getGMTValue().getAttribute('ng-reflect-value')).toBe('GMT-3.5');
+    });
+
+    it('should allow a half time zone to be set if it is present in the timezone array', async() => {
+        component.showTime = true;
+        component.showTimezone = true;
+
+        fixture.detectChanges();
+        await fixture.whenStable();
+
+        expect(getGMTValue().getAttribute('ng-reflect-value')).toBe('GMT');
+        expect(getGMTValue().getAttribute('ng-reflect-value')).toBe('GMT');
+    });
+
     function getDate(index: number = 0): HTMLElement | null {
         const headerSections = nativeElement.querySelectorAll('ux-date-range-picker .header-section');
         return headerSections[index].querySelector('.date-header');
     }
 
+    function getPicker(index: number = 0): HTMLElement | null {
```

Comment:
I think this will always only get the start 'ux-date-time-picker'. I'm not 100% on this though.